.env.example

```diff
@@ -1,2 +1,2 @@
-REACT_APP_BACKEND_API=
+REACT_APP_BACKEND_API=http://127.0.0.1:8000/accounts/github/login/
```

Comment:
should these be added to the `.env` file instead? seems like these are not loaded to `process.env`

Reply:
Yes. This should be added into .env file. I added this in .env.example for reference.